docs/server.rst

```diff
@@ -32,26 +32,80 @@ the broker.
 
 .. _history: https://github.com/wlanslovenija/tunneldigger/blob/master/HISTORY.rst
 
-Prerequisites
--------------
+Operating System
+----------------
 
 The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
-tunnels. The following modules are required for Tunneldigger operation:
+tunnels.
+
+These distributions are supported:
```

Comment:
It's not that tunneldigger doesn't work on other distributions, it's just that we don't have instructions. Maybe reword this to say that? Something like "The following instructions should work for these distributions:"

Reply:
26bd3b3 addresses this.